GLAA.Services/AccountCreation/AccountCreationPostDataHandler.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using AutoMapper;
+using GLAA.Domain.Models;
+using GLAA.Repository;
+using GLAA.ViewModels;
+using GLAA.ViewModels.LicenceApplication;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+
+namespace GLAA.Services.AccountCreation
+{
+    public class AccountCreationPostDataHandler : IAccountCreationPostDataHandler
+    {
+        private readonly IMapper mapper;
+        private readonly IEntityFrameworkRepository repository;
+        private readonly UserManager<GLAAUser> userManager;
+        private readonly IEmailService emailService;
+        private readonly IConfiguration configuration;
+
+        public AccountCreationPostDataHandler(IMapper mp, IEntityFrameworkRepository repo, UserManager<GLAAUser> um, IEmailService es, IConfiguration cs)
+        {
+            mapper = mp;
+            repository = repo;
+            userManager = um;
+            emailService = es;
+            configuration = cs;
+        }
+
+        public bool Exists(string email)
+        {
+            var user = userManager.FindByEmailAsync(email).GetAwaiter().GetResult();
```

Comment:
I still want to get to the bottom of this! I'm sure the async stuff should be fine to use!